tests/tool/tests/BUILD

```
@@ -11,10 +12,35 @@ py_test(
     ],
 )
 
+copy_file(
+    name = "appimage_runtime_copy_native",
+    src = select({
+        "@platforms//cpu:arm64": "@appimage_runtime_aarch64//file",
+        "@platforms//cpu:armv7e-m": "@appimage_runtime_armhf//file",
+        "@platforms//cpu:i386": "@appimage_runtime_i686//file",
+        "@platforms//cpu:x86_64": "@appimage_runtime_x86_64//file",
+    }),
+    out = "appimage_runtime_native",
+    allow_symlink = True,
+    is_executable = True,
+)
+
+copy_file(
+    name = "appimage_runtime_copy_x86_64",
+    src = "@appimage_runtime_x86_64//file",
+    out = "appimage_runtime_x86_64",
+    allow_symlink = True,
+    is_executable = True,
+)
+
 py_test(
     name = "test_mkappimage",
     size = "small",
     srcs = ["test_mkappimage.py"],
+    data = [
+        "appimage_runtime_x86_64",
+        ":appimage_runtime_native",
+    ],
```

I think this isn't used anymore with that test gone?
```suggestion
    data = [":appimage_runtime_native"],
```

---

Whoops, good catch